src/fu-quirks.h

```diff
@@ -168,7 +168,7 @@ gchar		*fu_quirks_lookup_by_guids		(FuQuirks	*self,
 
 /**
  * FU_QUIRKS_VENDOR:
- * @key: the USB device ID, e.g. `USB\VID_0763&PID_2806`
+ * @key: the USB device ID, e.g. `DeviceInstanceId=USB\VID_0763&PID_2806`
```

Comment:
Since key can be `Guid`, `Hwid`, or `DeviceInstanceId`, I think this needs a little more modification in documentation to show that.